tasks/kmt.py

```diff
@@ -628,7 +722,9 @@ def clean(ctx: Context, stack: Optional[str] = None, container=False, image=Fals
     }
 )
 def ssh_config(
-    _, stacks: Optional[str] = None, ddvm_rsa="~/dd/ami-builder/scripts/kernel-version-testing/files/ddvm_rsa"
+    ctx: Context,
+    stacks: Optional[str] = None,
+    ddvm_rsa="~/dd/ami-builder/scripts/kernel-version-testing/files/ddvm_rsa",
```

Comment:
Does this key exist in the agent repo at all? It would be nice to not require `ami-builder` to be cloned.

Reply:
It does, found it in `tasks/kernel_matrix_testing` too, changed the path.